content/en/tracing/profiling/getting_started.md

```diff
@@ -0,0 +1,209 @@
+---
+
+Profiling libraries are shipped within the following tracing language libraries. Select your language below to learn how to enable profiling for your application:
+
+{{< tabs >}}
+{{% tab "Java" %}}
+
+The Datadog Profiler requires [Java Flight Recorder][1]. The Datadog Profiling library is supported in OpenJDK 11+, Oracle Java 11+, and Zulu Java 8+ (minor version 1.8.0_212+). All JVM-based languages, such as Scala, Groovy, Kotlin, Clojure, etc. are supported. To begin profiling applications:
+
+1. If you are already using Datadog, upgrade your agent to version [7.20.2][2] or [6.20.2][2].
+
+2. Download `dd-java-agent.jar`, which contains the Java Agent class files:
+
+    ```shell
+    wget -O dd-java-agent.jar 'https://repository.sonatype.org/service/local/artifact/maven/redirect?r=central-proxy&g=com.datadoghq&a=dd-java-agent&v=LATEST'
+    ```
+
+     **Note**: Profiling is available in the `dd-java-agent.jar` library in versions 0.44+.
+
+3. Set `-Ddd.profiling.enabled` flag or `DD_PROFILING_ENABLED` environment variable to `true`. Update to your service invocation should look like:
+
+    ```diff
+    java -javaagent:dd-java-agent.jar -Ddd.profiling.enabled=true -jar <YOUR_SERVICE>.jar <YOUR_SERVICE_FLAGS>
+    ```
+
+**Note**:
+
+- The `-javaagent` needs to be run before the `-jar` file, adding it as a JVM option, not as an application argument. For more information, see the [Oracle documentation][3]:
+
+    ```shell
+    # Good:
+    java -javaagent:dd-java-agent.jar ... -jar my-service.jar -more-flags
+    # Bad:
+    java -jar my-service.jar -javaagent:dd-java-agent.jar ...
+    ```
+
+- It is strongly recommended to add tags like `service` or `version` as it provides the ability to slice and dice your profiles across these dimensions, enhancing your overall product experience. Use environment variables to set the parameters:
+
+| Environment variable                             | Type          | Description                                                                                      |
+| ------------------------------------------------ | ------------- | ------------------------------------------------------------------------------------------------ |
+| `DD_PROFILING_ENABLED`                           | Boolean       | Alternate for `-Ddd.profiling.enabled` argument. Set to `true` to enable profiling.               |
+| `DD_SERVICE`                                     | String        | The Datadog [service][2] name.     |
+| `DD_ENV`                                         | String        | The Datadog [environment][4] name, for example `production`.|
+| `DD_VERSION`                                     | String        | The version of your application.                             |
+| `DD_TAGS`                                        | String        | Tags to apply to an uploaded profile. Must be a list of `<key>:<value>` separated by commas such as: `layer:api, team:intake`.  |
```

Comment:
Can you double check we're ok with spaces after commas?

Reply:
easy to just remove the space hehe